scripts/install.sh

```diff
@@ -7,8 +7,15 @@
 
 set -u
 
+BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/router/releases/download"
+
+# Router version defined in apollo-router's Cargo.toml
+# Note: Change this line manually during the release steps.
+PACKAGE_VERSION="v0.16.0"
```

Comment:
could this be overridden by an environment variable or an argument?

Reply:
I mean, yes, but again, same comment as https://github.com/apollographql/router/pull/1433#discussion_r955845853, the point is that is not durable because the package installer might change over time based on the version.  It is very intentional that it does not allow installing different versions. 😸 

If you want a different version, you change the version in the `curl` install URL path itself.  That lets us track the actual invocations of the installer and change the installer over time with the versions.